docs/decisions/0002-use-changelog.rst

```diff
@@ -0,0 +1,70 @@
+2. Use CHANGELOG.rst
+====================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+This repository was using Github Releases only to capture changelog details, which has the following issues:
+
+* Additions and updates to the changelog don't go through PR review.
+* The changelog is not versioned with the repository, is not available with the repo documentation, cannot be seen in a single file, and is not available offline.
+
+Additionally, there was no guidance for formatting entries.
+
+Decision
+--------
+
+* Add a CHANGELOG.rst as the primary source of tracking changes.
+* The changelog will be formatted according to `keepachangelog.com`_.
+* Avoid redundancy in Github Releases.
+
+This resolves all issues noted under this ADR's `Context`_.
+
+Commit Message vs Changelog Entry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since changelog messages are for developers and consumers of your code, good changelog messages will often not match commit messages. Here are some examples:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Commit Message
+     - Changelog Entry
+   * - deps: update dependency some-dependency to 2.3.0
+     - No functional change
+
+       Note: This example changelog entry assumes there were no other changes for this version.
+   * - Fix SyntaxError in UserLogout class
+     - Fix 500 error during logout
+   * - **BREAKING CHANGE** Remove the dry_run parameter in the public foobarize API method. This parameter is deprecated in favour of the no_apply parameter. See docs for details.
+     - Rename dry_run parameter
```

Comment:
I think I would want to see **BREAKING CHANGE** in the changelog even more than in the commit. Maybe reverse these two?

Reply:
Same for next pair.